CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ The changelog for **SwifterSwift**. Also see the [releases](https://github.com/S
 - **UIView**
   - Added `addGestureRecognizers(_:)` which accepts an array of `UIGestureRecognizer` to add multiple gesture recognizers to a view with one call. [#523](https://github.com/SwifterSwift/SwifterSwift/pull/523) by [moyerr](https://github.com/moyerr)
   - Added `removeGestureRecognizers(_:)` which accepts an array of `UIGestureRecognizer` to remove multiple gesture recognizers from a view with one call. [#523](https://github.com/SwifterSwift/SwifterSwift/pull/523) by [moyerr](https://github.com/moyerr)
+- **UIViewController**
+  - Added `addChildViewController(_ child: , toContainerView:)` which accepts a UIViewController, to easily add child view controllers.
```

Comment:
Sorry for being pedantic here. It should be `addChildViewController(_:toContainerView)` and with backticks for `UIViewController` as well (in the line below too).

Reply:
I'm here to learn, feel free to destroy my pull request😂😂